signerverifier/rsa.go

```diff
@@ -99,7 +99,12 @@ func LoadRSAPSSKeyFromFile(path string) (*SSLibKey, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to load RSA key from file: %w", err)
 	}
+	return LoadRSAPSSKeyFromBytes(contents)
+}
 
+// LoadRSAPSSKeyFromFile returns an SSLibKey instance for an RSA key stored in a
```

Comment:
Can you add some information about the expected encoding of the key in the file?

Reply:
I have expanded on this, if their is something else you would like to add, tell me

Reply:
Could you indicate the RSA key is expected to be PEM encoded on disk?

Reply:
Added